runtime/src/bank.rs

```diff
@@ -143,6 +146,13 @@ impl Builtin {
     }
 }
 
+const CACHED_EXECUTOR_LIFETIME_EPOCHS: u64 = 5;
```

Comment:
@mvines Caching the executors for 5 epochs seem reasonable?  We don't need to accumulate a bunch over time, just the ones that are actively being used.  If a program is run that is not in the cache it will be re-cached for another 5 epochs.

Reply:
We could even be more aggressive to start with.  1 epoch seems good enough at first thought

Reply:
I switched to a cache with a fixed number of entries, went with 100 which should be about 10MB